providers/base/bin/capture-stressor.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+import subprocess
+
+CLASSES = 'cpu cpu-cache memory os pipe scheduler vm'
+
+classes_list = CLASSES.split()
```

Comment:
The first question is because the original **stress-ng-classes** is in the base provider, so I just kept it there.
However you are right, maybe we should move it to the resource provider. What do you think? @zongminl 

And the second question, I agree. It would make it clear.

Reply:
This is really a philosophical question, there is no technical requirement that resource jobs have to be in the resource provider.

The reason, as I see it, to maintain the resource provider would be that it could be considered a "base set" set of information that could be used in any number of template jobs etc contained in any number other of providers i.e. many jobs might like to filter on CPU type.

However, the stress-ng-stressors job is probably only of interest to this very specific set of jobs to help generate the -{stressor} jobs at run time and would not make sense outside of this specific category. Therefore, I don't see much need for it to move.